src/Inventory/Asset/Software.php

```diff
@@ -783,6 +787,26 @@ private function storeAssetLink()
                 $input['date_install']
             );
             $DB->executeStatement($stmt);
+
+            // log the new installation into software history
+            $version_name = $val->version ?? "";
+            $asset_name   = $this->item->fields['name'] ?? "";
```

Comment:
Both should be defined here.
```suggestion
            $version_name = $val->version;
            $asset_name   = $this->item->fields['name'];
```